### PR TITLE
Fix ResizeBilinear and ResizeNearestNeighbor and update related tests

### PR DESCRIPTION
**Pull request by cosineFish**
1) Fix tf ResizeBilinear and ResizeNearestNeighbor op with align_corners=True or half_pixel_centers=True 
2) Update related tests to make input data include x.5 and add tests for align_corners=True

### CODE REVIEW

**Comment by cosineFish** [comment]
This PR is ready for review but there are two failed checks.
Looks like the failed tests are not related to my change. Please help to re-run the failed tests, thanks!
```
tests/test_example.py FFF 
```

```
E               subprocess.CalledProcessError: Command '['python', '/home/vsts/work/1/s/tests/../examples/getting_started.py']' returned non-zero exit status 1.

../../../.conda/envs/tf2onnx/lib/python3.8/subprocess.py:516: CalledProcessError
```

File: tf2onnx/onnx_opset/nn.py [hunk 1]
```diff
@@ -1407,11 +1407,11 @@ def version_11(cls, ctx, node, **kwargs):
         nearest_mode = "floor"
         if "align_corners" in node.attr and node.attr["align_corners"].i:
             transformation_mode = "align_corners"
+            nearest_mode = "round_prefer_ceil"
         if "half_pixel_centers" in node.attr and node.attr["half_pixel_centers"].i:
             if node.type == "ResizeNearestNeighbor" and not ctx.is_target(constants.TARGET_TENSORRT):
                 # TensorRT only supports nearest_mode = "floor" for mode = "nearest"
-                transformation_mode = "half_pixel"
-                nearest_mode = "round_prefer_ceil"
+                transformation_mode = "tf_half_pixel_for_nn"
```
Comment (fatcat-z) [inline tf2onnx/onnx_opset/nn.py:1414]: Is this for debug?

**Comment by fatcat-z** [comment]
Please fix DCO issue.

File: tf2onnx/onnx_opset/nn.py (hunk 1 above)
Comment (cosineFish) [reply tf2onnx/onnx_opset/nn.py:1414]: Which name? tf_half_pixel_for_nn? There is no name for debug. All of them are valid attributes of onnx Resize op.
https://github.com/onnx/onnx/blob/main/docs/Changelog.md#resize-11

**Comment by cosineFish** [comment]
> Please fix DCO issue.

Fixed.

**Review by fatcat-z — APPROVED** [review]
LGTM, thank you for your contributions!

**Pull request merged**